lib/Dancer2/Config.pod

```diff
@@ -227,6 +227,54 @@ extension, and are text files, set the C<default_mime_type> to
 C<text/plain>.
 
 
+=head2 Serializing responses
+
+=head3 serializer (string)
```

Comment:
I can't speak to the content of the patch, so I'll be a doc Nazi and suggest that `(string)` should probably be replaced with something more perlish like ($string). Thanks!

Reply:
hello, it is really (string) and not $string.

You have to understand (string) has a datatype like per example (integer).
regards

sdeseille